### ISSUE TITLE
Bad ShapeCast / RayCast `enabled` property documentation

### ISSUE THREAD

**Issue by Swarkin** [godotengine/godot#87675]
### Tested versions

Applies to `ShapeCast*` and `RayCast*` docs, present on **all** versions.
https://docs.godotengine.org/en/latest/classes/class_shapecast3d.html

### System information

N/A (Documentation)

### Issue description

The enabled property of ShapeCast* / RayCast* is wrong:
> If true, collisions will be reported.

This is not true, as using `force_shapecast_update` does not require `enabled` to be set to true.
It should also include that having it enabled results in automatically recomputed data with each _physics_process tick, similar to https://docs.godotengine.org/en/latest/classes/class_shapecast3d.html#class-shapecast3d-method-force-shapecast-update.

### Steps to reproduce

N/A (Documentation)

### Minimal reproduction project (MRP)

N/A (Documentation)

**Comment by smix8** [comment]
That the `force_shapecast_update()` function ignores the `enabled` property is part of the linked documentation.

**Comment by AThousandShips** [comment]
The description isn't wrong, it says they will, the opposite isn't given, it doesn't say "only when true"

**Comment by Swarkin** [comment]
> That the `force_shapecast_update()` function ignores the `enabled` property is part of the linked documentation.

It should be made more clear in the `enabled` property docs as it currently is very vague.
If you have to read a single doc string multiple times to slowly be able to piece together its meaning by yourself, I think it should be changed, or at least point to said function to read that part as well

**Comment by AThousandShips** [comment]
I think it makes sufficiently clear in the method that is relevant, I don't see any reason to specify unrelated details in `enabled`, it's not an exception to the rule

**Comment by Swarkin** [comment]
What I would change is rephrase it to mention the fact that it recomputes new data every physics tick, as well as that force_*_update can be used to recompute data on the fly without needing the node to be enabled.

**Comment by Mickeon** [comment]
Feel free to open a PR with the changes you have suggested
